tmva/sofie/inc/TMVA/SOFIE_common.hxx

```diff
@@ -108,6 +108,46 @@ T* Unidirectional_broadcast(const T* original_data, const std::vector<size_t> or
 std::vector<size_t> Multidirectional_broadcast(std::vector<size_t> input1_shape, std::vector<size_t> input2_shape);
 std::string Clean_name(std::string input_tensor_name);
 
+template<typename T>
+T* BroadcastConvBias(const T* data, const size_t channel, const std::vector<size_t>& targetShape) {
+   size_t size = targetShape.size();
+   size_t index = 1;
+   if (targetShape[1] != channel) {
+      std::stringstream ss;
```

Comment:
This is missing an include right?

Reply:
Yes, I think so, thanks for finding this out. 
@axmat , can you please add an `#include <sstream>` ?
You can maybe add in the ConVTranspose PR